cypress/support/commands.ts

```diff
@@ -183,6 +183,28 @@ Cypress.Commands.add('keyboardControls', (triggerKeys: any = {}, count = 1) => {
   }
 });
 
+/**
+ * Intercept all requests and return
+ * @param {array} intercepts - Array of intercepts to return
+ * return {array} - Array of intercepted request strings
+ * return {string} - Intercepted request string
+ */
+Cypress.Commands.add('interceptAny', (method = '/GET/POST/PUT/PATCH/', url = '/v1/*', wait = false, timeout = 10000) => {
+  if (Array.isArray(url)) {
+    const interceptedUrls: string[] = [];
```

Comment:
This can be directly a map.
Assuming we are going to refactor this for an array case, we use `urls` plural.

```ts
const interceptedUrls: string[] = urls.map((url, i) => {
  cy.intercept(method, url).as(`interceptReq${ i }`);

  return `@interceptReq${ i }`
});
```